Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Editor/ForgeExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor.VersionControl;
+using UnityEngine;
+
+namespace BeardedManStudios.Forge.Networking.UnityEditor
```

Comment:
This namespace looks wrong. Shouldn't it be `BeardedManStudios.Editor`? 🤔

Reply:
Oh my bad. I'll correct that. It was auto-generated.

Reply:
Hmm, so that seems to be the namespace in mainline. I assume someone changed all the namespaces in dev.